pkg/sensors/program/loader_windows.go

```diff
@@ -0,0 +1,480 @@
+
+		return nil, fmt.Errorf("not supported on windows")
+	}
+}
+
+func TracepointAttach(load *Program, bpfDir string) AttachFunc {
+	return func(_ *ebpf.Collection, _ *ebpf.CollectionSpec,
+		prog *ebpf.Program, spec *ebpf.ProgramSpec) (unloader.Unloader, error) {
+
+		return nil, fmt.Errorf("not supported on windows")
+	}
+}
+
+func RawTracepointAttach(load *Program) AttachFunc {
+	return func(_ *ebpf.Collection, _ *ebpf.CollectionSpec,
+		prog *ebpf.Program, spec *ebpf.ProgramSpec) (unloader.Unloader, error) {
+
+		return nil, fmt.Errorf("not supported on windows")
+
+	}
+}
+
+func KprobeOpen(load *Program) OpenFunc {
+	return func(coll *ebpf.CollectionSpec) error {
+		return fmt.Errorf("not supported on windows")
+	}
+}
+
+func kprobeAttach(load *Program, prog *ebpf.Program, spec *ebpf.ProgramSpec,
+	symbol string, bpfDir string, extra ...string) (unloader.Unloader, error) {
+	return nil, fmt.Errorf("not supported on windows")
+}
+
+func windowsAttach(load *Program, prog *ebpf.Program, spec *ebpf.ProgramSpec,
+	symbol string, bpfDir string, extra ...string) (unloader.Unloader, error) {
+
+	link, err := link.AttachRawLink(link.RawLinkOptions{
+		Program: prog,
+		Attach:  ebpf.AttachWindowsProcess,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return unloader.ChainUnloader{
+		unloader.ProgUnloader{
+			Prog: prog,
+		},
+		unloader.LinkUnloader{
+			Link: link,
+		},
+	}, nil
+
+}
+
+func WindowsAttach(load *Program, bpfDir string) AttachFunc {
+	return func(coll *ebpf.Collection, collSpec *ebpf.CollectionSpec,
+		prog *ebpf.Program, spec *ebpf.ProgramSpec) (unloader.Unloader, error) {
+
+		return windowsAttach(load, prog, spec, load.Attach, bpfDir)
+	}
+}
+
+func KprobeAttach(load *Program, bpfDir string) AttachFunc {
+	return func(coll *ebpf.Collection, collSpec *ebpf.CollectionSpec,
+		prog *ebpf.Program, spec *ebpf.ProgramSpec) (unloader.Unloader, error) {
+
+		return kprobeAttach(load, prog, spec, load.Attach, bpfDir)
+	}
+}
+
+func UprobeAttach(load *Program) AttachFunc {
+	return func(_ *ebpf.Collection, _ *ebpf.CollectionSpec,
+		prog *ebpf.Program, spec *ebpf.ProgramSpec) (unloader.Unloader, error) {
+		return nil, fmt.Errorf("not supported on windows")
+
+	}
+}
+
+func MultiUprobeAttach(load *Program) AttachFunc {
+	return func(_ *ebpf.Collection, _ *ebpf.CollectionSpec,
+		prog *ebpf.Program, spec *ebpf.ProgramSpec) (unloader.Unloader, error) {
+		return nil, fmt.Errorf("not supported on windows")
+
+	}
+}
+
+func NoAttach() AttachFunc {
+	return func(_ *ebpf.Collection, _ *ebpf.CollectionSpec,
+		prog *ebpf.Program, _ *ebpf.ProgramSpec) (unloader.Unloader, error) {
+		return unloader.ChainUnloader{
+			unloader.ProgUnloader{
+				Prog: prog,
+			},
+		}, nil
+	}
+}
+
+func TracingAttach() AttachFunc {
+	return func(_ *ebpf.Collection, _ *ebpf.CollectionSpec,
+		prog *ebpf.Program, spec *ebpf.ProgramSpec) (unloader.Unloader, error) {
+		return nil, fmt.Errorf("not supported on windows")
```

Comment:
See previous comment.
In this case, I think we could even have a single `AttachFunc` that just returns a not supported error, and return it.